tests/dht/test_providers.nim

```diff
@@ -71,55 +72,66 @@ proc bootstrapNetwork(nodecount: int, rng = keys.newRng()) : seq[ProvidersProtoc
   #waitFor bootNode.bootstrap()  # immediate, since no bootnodes are defined above
 
   result = bootstrapNodes(nodecount - 1, @[bootnode.discovery.localNode.record], rng = rng)
-  result.insert(bootNode, 0)
+  result.insert((bootNode, privKey), 0)
```

Comment:
Use bootNodeKey here and remove lines 65-66?

Reply:
Done.